.github/workflows/grid_client_tests.yml

```diff
@@ -44,8 +44,9 @@ jobs:
           wget https://github.com/yggdrasil-network/yggdrasil-go/releases/download/v0.5.4/yggdrasil-0.5.4-amd64.deb
           sudo dpkg -i yggdrasil-0.5.4-amd64.deb
           sudo systemctl enable yggdrasil
-          PEERS=$(curl https://raw.githubusercontent.com/threefoldtech/zos-config/main/development.json | jq '.yggdrasil.peers[]' -r | sed 's/\//\\\//g' | sed -z 's/\n/\\n/g')
+          PEERS=$(curl https://raw.githubusercontent.com/AhmedHanafy725/yggdrasil-config/main/config.json | jq '.yggdrasil.peers[]' -r | sed 's/\//\\\//g' | sed -z 's/\n/\\n/g')
```

Comment:
Shouldn't this be under the organization account instead?

Reply:
it will be updated back when this issue is resolved https://github.com/threefoldtech/zos-config/issues/12